cluster/ssz.go

```diff
@@ -695,7 +697,7 @@ func hashRegistration(r Registration, hh ssz.HashWalker) error {
 	hh.PutUint64(uint64(r.GasLimit))
 
 	// Field (2) 'Timestamp' Bytes48
-	hh.PutBytes([]byte(r.Timestamp.String()))
+	hh.PutBytes([]byte(r.Timestamp.Format(time.RFC3339)))
```

Comment:
please use eth2 standard of ssz marshalling, also please ensure this hash equals canonical ssz hashing of validator registration

Reply:
Done 🫡